docs/user-guide/src/troubleshooting.md

```diff
@@ -0,0 +1,72 @@
+# Troubleshooting
+
+## Verify that Ironic and Baremetal Operator are healthy
+
+There is no point continuing before you have verified that the controllers are
+healthy. A "standard" deployment will have Ironic and Baremetal Operator running
+in the `baremetal-operator-system` namespace. Check that the containers are
+running, not restarting or crashing:
+
+```bash
+kubectl -n baremetal-operator-system get pods
+```
+
+Note: If you deploy Ironic outside of Kubernetes you will need to check on it in
+a different way.
+
+Healthy example output:
+
+```text
+NAME                                                     READY   STATUS    RESTARTS       AGE
+baremetal-operator-controller-manager-85b896f688-j27g5   1/1     Running   0              5m13s
+ironic-6bcdcb99f8-6ldlz                                  3/3     Running   1 (2m2s ago)   5m15s
+```
+
+(There has been one restart, but it is not constantly restarting.)
+
+Unhealthy example output:
+
+```text
+NAME                                                     READY   STATUS    RESTARTS      AGE
+baremetal-operator-controller-manager-85b896f688-j27g5   1/1     Running   0             3m35s
+ironic-6bcdcb99f8-6ldlz                                  1/3     Running   1 (24s ago)   3m37s
+```
+
+### Waiting for IP
+
+Make sure to check the logs also since Ironic may be stuck on "waiting for IP".
+
+```bash
+kubectl -n baremetal-operator-system logs deploy/ironic
```

Comment:
I suspect this won't work with IrSO... maybe should an example using the just shown pod name `ironic-6bcdcb99f8-6ldlz`?

More importantly, this section lacks any pointers on what to do about the "waiting for IP" problem (check networking in IrSO or wherever we normally have it?)

Reply:
Updated